configuration/boot_folder.md

```diff
@@ -47,6 +47,17 @@ This is the file to configure wireless network settings (if the hardware is capa
 
 There are various Device Tree blob files, which have the extension `.dtb`. These contain the hardware definitions of the various models of Raspberry Pi, and are used on boot to set up the kernel according to which Pi model is detected. More [details here](device-tree.md).
 
+### Kernel Files
+
+The boot folder will contain various kernel image files, used for the different Raspberry Pi models:
+
+| Filename | Processor | Raspberry Model | Notes |
+| ---------| ----------|-----------------|-------|
+| kernel.img | BCM2835 | Pi 0, Pi 1, Pi2 | |
```

Comment:
There were no Pi2s with the BCM2835 :stuck_out_tongue_winking_eye:

Reply:
Aaagh - yes remove it.